tests/runner/src/main/java/org/mule/test/runner/classloader/TestContainerClassLoaderFactory.java

```diff
@@ -43,7 +43,7 @@
   private final URL[] urls;
   private final URLClassLoader classLoader;
   private final DefaultModuleRepository testContainerModuleRepository;
-  private ArtifactClassLoader containerClassLoader;
+  private MuleArtifactClassLoader containerClassLoader;
```

Comment:
No, just a remnant of having initialize() in the parent class before going with the internal constructor approach.